pennylane/spin/lattice.py

```diff
@@ -72,6 +72,7 @@ def __init__(
         positions=None,
         boundary_condition=False,
         neighbour_order=1,
+        custom_edges=None,
```

Comment:
Missing docstring on custom_edges. I'm assuming it's a list of tuples though...?